### PR TITLE
Fix issues caused in the Safari browser

### PR DESCRIPTION
**Pull request by thivi**
Related https://github.com/wso2/product-is/issues/9784

### CODE REVIEW

File: oidc-js/src/client.ts [hunk 1]
```diff
@@ -44,6 +43,7 @@ import {
     sendRevokeTokenRequest
 } from "./utils";
 import { WebWorkerClient } from "./worker";
+import { isWebWorkerConfig } from "./helpers";
```
Comment (NipuniBhagya) [inline oidc-js/src/client.ts:46]: Check the import order.

File: oidc-js/src/helpers/typeguards.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NipuniBhagya) [inline oidc-js/src/helpers/typeguards.ts:2]:
```suggestion
 * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: oidc-js/src/helpers/typeguards.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ConfigInterface, WebWorkerConfigInterface } from "../models";
+import { Storage } from "../constants";
```
Comment (NipuniBhagya) [inline oidc-js/src/helpers/typeguards.ts:20]: Check the import order.

File: oidc-js/src/utils/sign-in.ts [hunk 4]
```diff
@@ -69,9 +69,9 @@ import {
     TokenRequestHeader,
     TokenResponseInterface,
     UserInfo,
-    WebWorkerConfigInterface,
-    isWebWorkerConfig
+    WebWorkerConfigInterface
 } from "../models";
+import { isWebWorkerConfig } from "../helpers";
```
Comment (NipuniBhagya) [inline oidc-js/src/utils/sign-in.ts:74]: This import should come before the ../models import

**Review by NipuniBhagya — APPROVED** [review]

File: oidc-js/src/helpers/typeguards.ts (hunk 2 above)
Comment (thivi) [reply oidc-js/src/helpers/typeguards.ts]: Resolved by 3ff130e

File: oidc-js/src/utils/sign-in.ts (hunk 4 above)
Comment (thivi) [reply oidc-js/src/utils/sign-in.ts]: Resolved by a0897b4

File: oidc-js/src/helpers/typeguards.ts (hunk 3 above)
Comment (thivi) [reply oidc-js/src/helpers/typeguards.ts:19]: Resolved by a0897b4

File: oidc-js/src/client.ts (hunk 1 above)
Comment (thivi) [reply oidc-js/src/client.ts]: Resolved by a0897b4

**Pull request merged**